.github/workflows/2_bump_versions.yml

```
@@ -0,0 +1,62 @@
+name: Bump versions
+
+on:
+  workflow_dispatch:
+    inputs:
+      npm_channel:
+        description: 'Prisma CLI NPM Channel'
+        required: true
+      version:
+        description: 'New Prisma CLI version in that channel'
+        required: true
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Install Semver # Needed to use semver in scripts
+        run: npm install
+      - name: Setup Git
+        run: |
+          git config --global user.email "prismabots@gmail.com"
+          git config --global user.name "Prismo"
+      - name: Update versions in scripts folder 
+        run: |
+          echo "${{ github.event.inputs.version }}" >scripts/versions/prisma_${{ github.event.inputs.npm_channel }}
+      - name: Update next extension version in scripts folder
+        id: update # Used to access the calculated next extension version in later steps
+        run: node scripts/bump_extension_version_in_scripts.js ${{ github.event.inputs.npm_channel }}
+      - name: Push to master
+        run: |
+          git add -A .
+          git commit -m "bump Prisma CLI ${{ github.event.inputs.npm_channel }} version to ${{ github.event.inputs.version }} and extension version to ${{ steps.update.next_extension_version }} in scripts."
+          git pull --rebase
+          git push
+      - name: Switch branch 
```

```suggestion

      - name: Switch branch 
```

How do you feel about using empty lines to separate "blocks" of the jobs? This is highly subjective, but I think figuring our a "pattern" that we can use to get a bit more structure in the workflows might be worth it.